File: tuts/003-s3-gettingstarted/ruby/scenario_getting_started.rb
# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
# SPDX-License-Identifier: Apache-2.0

require_relative 's3_wrapper'

def run_scenario
  client = Aws::S3::Client.new
  wrapper = S3Wrapper.new(client)
  puts "Running S3 getting started scenario..."
  # TODO: setup, interact, teardown
  puts "Scenario complete."
end

run_scenario if __FILE__ == $PROGRAM_NAME

File: tuts/003-s3-gettingstarted/python/scenario_getting_started.py
# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
# SPDX-License-Identifier: Apache-2.0

import uuid
import boto3
from s3_wrapper import S3Wrapper


def run_scenario():
    client = boto3.client("s3")
    wrapper = S3Wrapper(client)
    suffix = uuid.uuid4().hex[:8]
    print(f"Running S3 getting started scenario...")
    # TODO: implement setup, interact, teardown
    print("Scenario complete.")


if __name__ == "__main__":
    run_scenario()
